openfisca_us/variables/gov/states/in/tax/income/exemptions/in_qualifying_child_count.py

```diff
@@ -5,7 +5,6 @@ class in_qualifying_child_count(Variable):
     value_type = int
     entity = TaxUnit
     label = "IN qualifying depdendent child count"
```

Comment:
We can have arbitrary strings here.
```suggestion
    label = "IN qualifying dependent child count"
    unit = "child"
```